src/api/species/species.ts

```diff
@@ -276,11 +277,10 @@ export async function acceptProblemSuggestion(problemId: number) {
   const response: UpdateSpeciesResponse = {
     requestSucceeded: true,
   };
-  try {
-    await axios.post(SPECIES_PROBLEM.replace('{problemId}', problemId.toString()));
-  } catch {
+  await axios.post(SPECIES_PROBLEM.replace('{problemId}', problemId.toString())).catch((e) => {
```

Comment:
Not sure why I missed this.. Try/catch would be the better pattern with await.. curious why the promise like handling.

Reply:
one is an asynchronous pattern
```
  return promise.then().catch();
```
the other is a blocking pattern
```
  try {
     return await promise;
  } catch(e) {
     return error
  }
```
there is no hard and fast rule... its mostly for readability.
you can mix/match them for sure but no point in using await..
plenty of references online .. hate pointing to links from random sources but we all like stackoverflow.. so here you go https://stackoverflow.com/questions/69362121/when-should-i-use-try-catch-instead-of-then-catch
once again, its a matter of readability versus theoretical correctness.

Reply:
fwiw  - before introduction of `await` a lot of us were in promise chain hell.. hard to read/follow/debug code...

Reply:
Thank you. I will make the change ❤️